src/commands/CreateCommand.ts

```diff
@@ -44,6 +46,11 @@ export class CreateCommand extends BaseCommand<CreateCommandArgs> {
         description:
           "The client secret of the OAuth2 client that should be used to authenticate the Connector on the Backbone. Defaults to the value you specified during 'cman init'.",
       })
+      .option("additional-configuration", {
```

Comment:
You should maybe add a yargs-example for how to use the new flag (in line 64).

Reply:
yep